lib/msal-core/src/telemetry/TelemetryConstants.ts

```diff
@@ -0,0 +1,19 @@
+export const EVENT_NAME_PREFIX: String = "msal.";
```

Comment:
I like the code style here, but we should match our constants format across the project. 

Also should we start splitting constants up by usage across other files as well?

Reply:
Tbd how we split up, but I dont want to add to the monster const file we have, so I am modularizing for telemetry. Also there is a good chance this code will be modularized and reused for node as well and so I don't want to have to rip it out in the future again.

Reply:
We should probably stop growing that file at some point then, and try to separate it out so we can have better control and readability. Maybe a constants subdirectory or something.

Reply:
yep big fan of that, as we modularize, we should namespace constants to their own files / objects